front-end/src/pages/Courses.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Divider from "@mui/material/Divider";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import Typography from "@mui/material/Typography";
+import courseList from "../test-data/courseList";
```

Comment:
test-data folder seems not to exist .ie the  import courseList from "../test-data/courseList"; seems not to be a valid import